components/03-modules/active-filters/mocks/active-filters.json

```diff
@@ -1,39 +1,32 @@
-{
-  "clear": {
+[
+  {
+    "label": "Label",
+    "value": "Value",
+    "ariaLabel": "Disable filter",
     "href": "#",
-    "text": "Clear all",
-    "title": "Clear all filters"
+    "icon": {
+      "iconId": "close",
+      "iconTitle": "Close"
+    }
   },
-  "filters": [
-    {
-      "label": "Label",
-      "value": "Value",
-      "ariaLabel": "Disable filter",
-      "href": "#",
-      "icon": {
-        "iconId": "close",
-        "iconTitle": "Close"
-      }
-    },
-    {
-      "label": "Label2",
-      "value": "Value2",
-      "ariaLabel": "Disable filter",
-      "href": "#",
-      "icon": {
-        "iconId": "close",
-        "iconTitle": "Close"
-      }
-    },
-    {
-      "label": "Label3",
-      "value": "Value3",
-      "ariaLabel": "Disable filter",
-      "href": "#",
-      "icon": {
-        "iconId": "close",
-        "iconTitle": "Close"
-      }
+  {
+    "label": "Label2",
+    "value": "Value2",
+    "ariaLabel": "Disable filter",
+    "href": "#",
+    "icon": {
+      "iconId": "close",
+      "iconTitle": "Close"
+    }
+  },
+  {
+    "label": "Label3",
+    "value": "Value3",
+    "ariaLabel": "Disable filter",
+    "href": "#",
+    "icon": {
```

Comment:
close icon shouldn't be defined in mock, we can pass it to the component as an option